### PR TITLE
safari light fix

### PR DESCRIPTION
**Pull request by outsharded**
fixed the safari banner for light mode

### CODE REVIEW

**Comment by seidnerj** [comment]
@tecdude, try changing the tests to run on "windows-2019" instead of "windows-latest", this helped me fix the timeouts in the windows builds.

**Comment by tecdude** [comment]
How can I change it?

**Comment by seidnerj** [comment]
You can change this by editing the file ".github/workflows/test.yml" and changing the YML file so that everywhere where it says "- os: windows-latest" it would now say "- os: windows-2019".

**Comment by tecdude** [comment]
updated

**Comment by seidnerj** [comment]
Hurray!

**Comment by tecdude** [comment]
These will make some interface elements more human'-readable and user friendly for people with little technical experience

**Comment by donavanbecker** [comment]
@tectrainguy Once the commented lines are reverted, please rebase to beta branch.

**Comment by bwp91** [comment]
was there any reason why this wasn't merged? @donavanbecker

File: ui/src/app/modules/restart/restart.component.html [hunk 1]
```diff
@@ -2,14 +2,14 @@
   <div class="col-md-12 text-center">
     <h3 class="primary-text" [translate]="'restart.title_restart'">Restarting Homebridge</h3>
     <p class="grey-text" *ngIf="!timeout && !error" [translate]="'restart.message_please_wait_while_server_restarts'">
-      Please wait, this page will automatically redirect when the server is back online.</p>
+      Please wait, this page will automatically redirect to the control panel when the server is back online.</p>
```
Comment (bwp91) [inline ui/src/app/modules/restart/restart.component.html:5]: are these translations not in a file somewhere?

**Comment by bwp91** [comment]
@outsharded could you show a screenshot of exactly what this fixes? ideally a before and after? would be grateful!

**Pull request closed without merging**